src/test/java/seedu/taassist/logic/commands/AddsCommandTest.java

```diff
@@ -65,6 +77,66 @@ public void execute_validSessionsWithDuplicates_success() throws CommandExceptio
                 commandResult.getFeedbackToUser());
     }
 
+    //==================================== Integration Tests =========================================================
+
+    @BeforeEach
+    public void enterFocusMode() {
+        ModuleClass moduleClassToFocus = model.getModuleClassList().get(0);
+        model.enterFocusMode(moduleClassToFocus);
+        expectedModel.enterFocusMode(moduleClassToFocus);
+    }
+
+    @Test
+    public void execute_addSingleNewSession_success() throws CommandException {
+        ModuleClass focusedClass = model.getFocusedClass();
+        Session validSession = new SessionBuilder().withName(VALID_SESSION_LAB1).build();
+        assert !focusedClass.hasSession(validSession);
+
+        Set<Session> validSessions = new HashSet<>(List.of(validSession));
+        AddsCommand command = new AddsCommand(validSessions);
+        command.execute(expectedModel);
+
+        assertCommandSuccess(command, model, AddsCommand.getCommandMessage(validSessions, new HashSet<>()),
```

Comment:
Should `getCommandMessage` have public visibility? I'd prefer if a hardcoded message is constructed here and the method is set to have private visibility.